### DISCUSSION TITLE
ts-loader vs esbuild-loader

### DISCUSSION THREAD

**Discussion by inyeong-kang** [woowacourse-teams/2023-votogether#490] [자유로운 토론]
esbuild-loader로 마이그레이션해보고 싶어요!
그 이유는 아래와 같습니다.

1. 프로덕션 빌드 시간 절감 (esbuild loader를 쓰면, 플러그인들을 간소화할뿐 아니라 빌드 시간을 줄여 개발 효율을 올릴 수 있다고 생각해요😃)
2. dev-server 실행 시간 대폭 절감 (m1 칩이 없는 개발 환경에서는.. 3배로 빠르게 실행 가능할거 같네요)

**Comment by Gilpop8663** [comment]
빌드 시간을 많게는 3배 이상 시간을 단축시킬수 있다고 하여 개발 속도 향상이 체감될 것이라고 생각들어서 찬성입니다

그리고 loader에 대해 더 많이 알아갈 수 있다는 점도 있어요

<img width="1111" alt="image" src="https://github.com/woowacourse-teams/2023-votogether/assets/80146176/85e2a8fc-721d-4269-885c-57a972d27279">


https://fe-developers.kakaoent.com/2022/220707-webpack-esbuild-loader/
